components/gitpod-db/src/typeorm/workspace-db-impl.spec.db.ts

```diff
@@ -64,6 +66,15 @@ export class WorkspaceSpec {
         ];
         const errors = [undefined, undefined, undefined, undefined, "failed", undefined, undefined];
 
+        await this.projectDB.storeProject({
```

Comment:
[nit] since we will omit deleted Repo's prebuilds, will it be good if we could add a db test case for it?

Reply:
Added tests in https://github.com/gitpod-io/gitpod/pull/19558/commits/11133b775971dac512e89f9a0aaec1f7f594f29e